src/components/pages/devices/devicesGrid/devicesGrid.js

```diff
@@ -86,11 +115,16 @@ export class DevicesGrid extends Component {
         t: this.props.t
       },
       /* Grid Events */
+      onSoftSelectChange: this.onSoftSelectChange,
       onHardSelectChange: this.onHardSelectChange,
       onGridReady: this.onGridReady
     };
-    return (
-      <PcsGrid {...gridProps} />
-    );
+    const openFlyout = (this.state.openFlyoutName === 'delete')
+      ? <DeviceDeleteContainer key="device-flyout-key" onClose={this.closeFlyout} devices={this.deviceGridApi.getSelectedRows()} />
+      : undefined;
```

Comment:
I typically return `null` instead of `undefined`. `undefined` will still work off course but I prefer the semantics of  `null`. The reason is that `null` is a value that is a "non-value" where as `undefined` says there is not value at all. It seems more reasonable to have an array where all values are `defined` but some are `null` rather than some values are `undefined`.